src/css/components/messages.css

```diff
@@ -0,0 +1,68 @@
+.messages__wrapper {
+  .messages {
+    grid-column: 1 / 7;
+    align-items: center;
+    background: var(--color--gray-0);
+    border-radius: 0 2px 2px 0;
+    margin: 72px 0;
+    padding: 22px 27px 23px;
```

Comment:
Vertical padding within the design is 20px, but let's change that to one of our vertical rhythm units. So, a `padding: var(--sp)` should be sufficient.